src/databricks/labs/ucx/source_code/linters/context.py

```diff
@@ -21,7 +21,13 @@
 
 
 class LinterContext:
-    def __init__(self, index: MigrationIndex | None = None, session_state: CurrentSessionState | None = None):
+    def __init__(
+        self,
+        index: MigrationIndex | None = None,
+        session_state: CurrentSessionState | None = None,
+        dbr_version: tuple[int, int] | None = None,
```

Comment:
make dbr version a property of `CurrentSessionState` as well. we'll have few more and we should have just one container to pass along, not many parameters we may forget about.